packages/@ember/debug/data-adapter.ts

```diff
@@ -544,22 +544,22 @@ export default class DataAdapter<T> extends EmberObject {
     @method getModelTypes
     @return {Array} Array of model types.
   */
-  getModelTypes(): NativeArray<{ klass: unknown; name: string }> {
+  getModelTypes(): Array<{ klass: unknown; name: string }> {
     let containerDebugAdapter = this.containerDebugAdapter;
 
     let stringTypes = containerDebugAdapter.canCatalogEntriesByType('model')
       ? containerDebugAdapter.catalogEntriesByType('model')
       : this._getObjectsOnNamespaces();
 
     // New adapters return strings instead of classes.
-    let klassTypes = emberA(stringTypes).map((name) => {
+    let klassTypes = stringTypes.map((name) => {
```

Comment:
These uses of `emberA` were unnecessary since `map` and `filter` are native methods.

Reply:
On the one hand, this is *definitely* a change I'm excited to be able to make; on the other, my note here is the same as above. Let's do it in a separate PR so it's easy to trace back to if for some reason it ends up being an issue for folks.